.github/workflows/validate-new-issue-test.yml

```diff
@@ -192,3 +192,34 @@ jobs:
           .github/workflows/test-helpers/get-issue-validation-log "${issue_number}"
           set -x
           grep "Closed with: .* created an empty issue" log
+  case_8:
+    name: "It labels validated issues with `Status: Unrouted`."
+    needs: case_7
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - id: setup
+        name: setup
+        env:
+          GITHUB_TOKEN: ${{ secrets.PAT_NONMEMBER }}
+        run: |
+          issue_number=$(basename $(gh issue create --title "Greetings, program!" --body "
+          ### Environment
+          ### Steps to Reproduce
+          ### Expected Result
+          ### Actual Result
+          "))
+          echo "::set-output name=issue_number::${issue_number}"
+      - name: test
+        run: |
+          issue_number="${{ steps.setup.outputs.issue_number }}"
+          .github/workflows/test-helpers/get-issue-validation-log "${issue_number}"
```

Comment:
So we are still using debug logs to test things? I mean this one should be pretty simple, just check whether the issue has that label or not?
```shell
gh issues view $issue_number | grep "^Labels:"
``

should come close, no?

Reply:
> So we are still using debug logs to test things?

Not really. Imagine that `get-issue-validation-log` is renamed to `wait-for-issue`, as it is in fact renamed on #46. :)

> gh issues view $issue_number  | grep "^Labels:"

See comment on #46 at https://github.com/getsentry/.github/pull/46/files#diff-300cc84d45c813ed2972f3ea7f5e093e0f99dda06427a2b70bfac39c0a2d0470R7-R9. I had trouble using `gh issues view` in a pipeline, hence the reliance on JSON output there and here.